api/prontolista/testcases/tests/test_models.py

```diff
@@ -24,3 +24,10 @@ def test_model_should_have_foreign_key_to_project(self):
         assert actual.project.name == expected_project_name
         assert actual.created
         assert actual.modified
+
+    def test_model_should_get_name_as_string_representation(self):
+        project = mommy.make(Project)
```

Comment:
Shall we start migrate to use model_bakery here too?

Reply:
Done.